mkosi/distributions/none.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import urllib.parse
+from collections.abc import Sequence
+
+from mkosi.architecture import Architecture
+from mkosi.distributions import Distribution, DistributionInstaller, PackageType
+from mkosi.log import die
+from mkosi.state import MkosiState
+
+
+class NoneInstaller(DistributionInstaller):
+    @classmethod
+    def filesystem(cls) -> str:
+        return "ext4"
+
+    @classmethod
+    def package_type(cls) -> PackageType:
+        return PackageType.rpm
```

Comment:
Let's add `none` to `PackageType` and use it here

Reply:
Done